velodrome/prober/blackbox.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: blackbox-prober
+  labels:
+    app: blackbox-prober
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: blackbox-prober
+    spec:
+      containers:
```

Comment:
I think this container needs to expose port 9115 for the service resource.

Reply:
I've made the service an internal LB instead, because we have no plans to make this accessible from outside of the cluster.

If you recall, doing so would be a serious security hazard.